force-app/main/default/lwc/attendance/attendance.js

```diff
@@ -384,10 +383,20 @@ export default class Attendance extends NavigationMixin(LightningElement) {
     }
 
     handleCancel() {
-        this.serviceDeliveries = this.serviceDeliveries.map(a => ({ ...a }));
+        this.initialServiceDeliveries();
+        this.sortServiceDeliveries();
```

Comment:
Not following why we need to sort on cancel.  I tried commenting out this line and it seemed to be fine.  Curious what I might be missing.